libbeat/docs/runconfig.asciidoc

```diff
@@ -11,7 +11,7 @@
 //////////////////////////////////////////////////////////////////////////
 
 [[configuration-run-options]]
-=== Run Options (Optional)
+=== Run Options Configuration (Optional)
```

Comment:
Yes, they are not in the config file (but probably should be). I actually checked the source code and these options are part of packetbeat and not libbeat. Lets move it to packetbeat and in a second step either remove them if not needed or add them to the packetbeat full config file (with tests ...)